src/screens/ProfileScreen/index.js

```diff
@@ -268,10 +269,9 @@ const ProfileScreen = ({route}) => {
     try {
       setIsFetchingList(true);
       setIsHitApiFirstTime(true);
-      const cacheFeed = StorageUtils.myFeeds.get();
       const result = await getSelfFeedsInProfile(offset, limit);
       const {data: dataMyFeed} = result;
-      const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
+      const mapNewData = mappingColorFeed(dataMyFeed);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `mappingColorFeed` function is now being called with only one argument, whereas previously it was called with an object containing two properties: `dataFeed` and `dataCache`. If the function still expects these two properties, this will cause an error. Please ensure that the function signature of `mappingColorFeed` has been updated to reflect this change.
```diff
- const cacheFeed = StorageUtils.myFeeds.get();
- const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
+ const mapNewData = mappingColorFeed(dataMyFeed);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->